### PR TITLE
inline CreateConfigMap into the places where it is used.

### LINKED ISSUE #11436: test/utils.go: Inline CreateConfigMap
**Is your feature request related to a problem? Please describe**:

The function `CreateConfigMap` in tests/utils.go could be inlined, since it does not provide much additional value. Also since this is test code, assertions should be used instead of panicking on errors.

**Describe the solution you'd like**:

Inline `CreateConfigMap` into the places where it is used.

### PR DESCRIPTION
**Pull request by anishbista60**
### What this PR does
Before this PR:

The functions `CreateConfigMap` in `tests/utils.go` doesnot look so useful because these function being called again and again in `tests/config_test.go`.

After this PR:

 This function are removed from `tests/utils.go` and inlined into the `tests/config_test.go` where they are used.

Fixes #11436

### Why we need it and why it was done in this way
- Because it removes the overhead of the function call itself. 
- The second is it permits the compiler to more effectively apply other optimisation strategies.

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
None
```

### CODE REVIEW

File: tests/config_test.go [hunk 1]
```diff
@@ -141,15 +152,23 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
...
 			})
 
 			AfterEach(func() {
 				for _, configMap := range configMaps {
-					tests.DeleteConfigMap(configMap, testsuite.GetTestNamespace(nil))
+					if err := kubevirt.Client().CoreV1().ConfigMaps(testsuite.GetTestNamespace(nil)).Delete(context.Background(), configMap, metav1.DeleteOptions{}); err != nil {
+						Expect(err).To(MatchError(errors.IsNotFound, "IsNotFound"))
+					}
 				}
+
```
Comment (alicefr) [inline tests/config_test.go:171]: nit:newline

**Comment by alicefr** [comment]
/ok-to-test

The code looks good to me.
/cc @0xFelix

**Comment by anishbista60** [comment]
@alicefr yeah , good to go

**Comment by 0xFelix** [comment]
Also please run `make format` over this, the formatting looks off in some places.

**Pull request closed without merging**